drivers/gnss/gnss_quectel_lcx6g.c

```diff
@@ -50,7 +50,7 @@ struct quectel_lcx6g_data {
 	/* UART backend */
 	struct modem_pipe *uart_pipe;
 	struct modem_backend_uart uart_backend;
-	uint8_t uart_backend_receive_buf[128];
+	uint8_t uart_backend_receive_buf[126];
```

Comment:
this change looks off, is it intended?

Reply:
This specific change was not intended, but increasing the default buffer size to 256 with the Kconfig is :)

Reply:
I updated the commit message to state the intentional increase in buffer size :)